hexrd/ui/mask_manager_dialog.py

```diff
@@ -84,8 +84,8 @@ def setup_connections(self):
         self.ui.masks_table.cellDoubleClicked.connect(self.get_old_name)
         self.ui.masks_table.cellChanged.connect(self.update_mask_name)
         self.ui.masks_table.customContextMenuRequested.connect(
-            self.context_menu_event)
-        self.ui.export_masks.clicked.connect(self.export_visible_masks)
+            self.export_masks)
```

Comment:
I think it makes more sense for `self.context_menu_event()` to call `export_masks()` rather than `self.export_masks()` to call the context menu event.